src/main/java/bio/terra/cli/command/resource/Resolve.java

```diff
@@ -98,20 +103,71 @@ protected void execute() {
   }
 
   private void printText(JSONObject resourceNamesToCloudIds) {
-    // No need to print the resource name as well as the cloudId if there's only one resource.
-    boolean printResourceName = resourceNamesToCloudIds.length() > 1;
-    for (var resourceName : resourceNamesToCloudIds.keySet()) {
-      if (printResourceName) {
-        OUT.println(resourceName + ": " + resourceNamesToCloudIds.get((String) resourceName));
-      } else {
-        OUT.println(resourceNamesToCloudIds.get((String) resourceName));
-      }
+    // For a single resource, just print cloud ID (no resource name)
+    if (resourceNamesToCloudIds.length() == 1) {
+      String resourceName = (String) resourceNamesToCloudIds.keySet().iterator().next();
+      OUT.println(resourceNamesToCloudIds.get(resourceName));
```

Comment:
return here (or put the logic below inside `else` block). as-is this prints single resources twice, once with just the cloud ID then again as a table

Reply:
Oops done, thanks